drivers/usb/usbccgp/misc.c

```diff
@@ -95,17 +95,8 @@ AllocateItem(
     IN POOL_TYPE PoolType,
     IN ULONG ItemSize)
 {
-    /* Allocate item */
-    PVOID Item = ExAllocatePoolWithTag(PoolType, ItemSize, USBCCPG_TAG);
-
-    if (Item)
-    {
-        /* Zero item */
-        RtlZeroMemory(Item, ItemSize);
-    }
-
-    /* Return element */
-    return Item;
+    /* Allocate, zero and return item */
```

Comment:
```suggestion
```
A captain-obvious comment. You didn't comment it in bdasup.c, so why doing it here? The functions name is self-explaining.

Reply:
I agree, but I leave the comments as/where they are (format and content).